tests/tests/tier0/bluechi-agent-user-bus/main.fmf

```diff
@@ -1,3 +1,3 @@
 summary: Test agent connecting to user bus
 id: 6886e7c8-7548-4cf7-8dd4-d2d46b125fd3
-tag: [multihost] 
```

Comment:
I think we need to remove the `testing-farm-container` tag now. It was used only to exclude this test from running in testing farm.

Reply:
Honestly, I didn't know where this tag was originating from. Lets keep it for this MR, but add it to the AC of #900 - if its possible to completely remove or enable this test again only for GH CI.

Reply:
Added a note to the Issue description, and I see that you have added a comment. Hopefully we will not forget and not leave it dangling there :)